sendgrid/sendgrid.py

```diff
@@ -58,3 +57,14 @@ def api_key(self, value):
     @property
     def impersonate_subuser(self):
         return self._impersonate_subuser
+
+
+class SendGridClient(SendGridAPIClient):

```

Comment:
I think having `SendGridAPIClient` and `SendGridClient` will cause confusion. I suggest we add the function to the `SendGridAPIClient` class. Maybe that particular function belongs in the helper, like [this](https://github.com/sendgrid/sendgrid-csharp/blob/master/src/SendGrid/Helpers/Mail/MailHelper.cs#L22). It would be called `send_single_email` or similar.